packages/appChrome/components/AppChrome.tsx

```diff
@@ -6,7 +6,7 @@ import {
   flexItem,
   textSize,
   flush
-} from "../../shared/styles/styleUtils";
+} from "@dcos/ui-kit-shared/dist/styles/styleUtils";
```

Comment:
💥 I thought we wanted to get rid of the sublevel imports for the styleUtils, did I misunderstand our intentions here?